src/app/Scenes/SellWithArtsy/ArtworkForm/Components/SubmitArtworkAdditionalDocuments.tsx

```diff
@@ -62,9 +214,128 @@ export const SubmitArtworkAdditionalDocuments = () => {
             </Text>
 
             <Spacer y={2} />
+
+            <Button block variant="outline" onPress={handleUpload}>
+              Add Documents
+            </Button>
+
+            <Separator my={2} borderColor="black10" />
+
+            <Flex rowGap={space(2)}>
+              {values.additionalDocuments.map((document) => {
+                return (
+                  <UploadedFile
+                    key={document.id}
+                    document={document}
+                    progress={progress[document.id]}
+                    onRemove={() => {
+                      handleDelete(document)
+                    }}
+                  />
+                )
+              })}
+            </Flex>
           </Flex>
         </ScrollView>
       </Flex>
     </ProvideScreenTrackingWithCohesionSchema>
   )
 }
+
+const CONTAINER_HEIGHT = 60
+
+const UploadedFile: React.FC<{
+  document: NormalizedDocument
+  onRemove: () => void
+  progress?: number | null
+}> = ({ document, onRemove, progress }) => {
+  const space = useSpace()
+  const sizeInMb = document.size ? (parseFloat(document.size) / 1024 / 1024).toFixed(2) : 0
+
+  return (
+    <Flex>
+      <Flex
```

Comment:
Can this outer flex be removed?

```suggestion
  <Flex
```

Reply:
yess